docs/src/pages/demos/breadcrumbs/CollapsedBreadcrumbs.js

```diff
@@ -27,7 +27,7 @@ function CollapsedBreadcrumbs(props) {
   const { classes } = props;
 
   return (
-    <Paper className={classes.paper}>
+    <Paper elevation={0} className={classes.paper}>
```

Comment:
I'm kind of sick of this elevation shadow everywhere. In the breadcrumb case, I think that it creates too much distraction.

Reply:
👍